### DISCUSSION TITLE
組み立て

### DISCUSSION THREAD

**Discussion by na260** [Yowkees/keyball#63] [Q&A] [answered]
組み立てていただいた状態で発送いただくことは可能なのでしょうか？

**Comment by Yowkees** [comment, marked as answer]
今年12月頃より組み立てサービスを開始予定です。
ご検討お願いいたします。

**Reply by na260** [reply]
ご回答ありがとうございます。
12月だとちょっと使いたい時期に間に合わないので自力で組み立てます。

**Reply by Yowkees** [reply]
ご期待に添えず申し訳ありませんでした💦
はんだ付けについてご不明点等あればご連絡ください
